sklearn/preprocessing/tests/test_data.py

```diff
@@ -2000,6 +2003,20 @@ def test_binarizer(constructor):
             binarizer.transform(constructor(X))
 
 
+@pytest.mark.parametrize(
+    "array_namespace, device, dtype_name", yield_namespace_device_dtype_combinations()
+)
+def test_binarizer_array_api_int(array_namespace, device, dtype_name):
```

Comment:
Could you add a concise docstring specifying what this additional test is checking for?